src/graphnet/pisa/fitting.py

```diff
@@ -15,6 +15,7 @@
 import io
 from configupdater import ConfigUpdater
 from contextlib import contextmanager
+from graphnet.data.utils import run_sql_code, save_to_sql
```

Comment:
Due to refactor in #215 
```suggestion
from graphnet.data.sqlite import run_sql_code, save_to_sql
```